src/main/java/com/minecolonies/coremod/blocks/cactus/BlockCactusStair.java

```diff
@@ -0,0 +1,20 @@
+package com.minecolonies.coremod.blocks.cactus;
+
+import com.minecolonies.api.util.constant.Constants;
+import com.minecolonies.coremod.blocks.AbstractBlockMinecoloniesStairs;
+import com.minecolonies.coremod.creativetab.ModCreativeTabs;
+import net.minecraft.block.SoundType;
+import net.minecraft.block.state.IBlockState;
+
+import java.util.Locale;
+
+public class BlockCactusStair extends AbstractBlockMinecoloniesStairs<BlockCactusStair> {
+
+    public BlockCactusStair(IBlockState modelState) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)